backup.sh

```diff
@@ -95,7 +95,7 @@ if [[ $DB_ENGINE == "sqlserver-se" ]]; then
   echo "Start the Mssql backup..."
   TASK_OUTPUT=$(sqlcmd_with_backoff $SQLCMD -S $RDS_ENDPOINT -U $RDS_USERNAME -P $RDS_PASSWORD -Q \
     "exec msdb.dbo.rds_backup_database @source_db_name='$DB_NAME', \
-    @s3_arn_to_backup_to='arn:aws:s3:::$BACKUP_BUCKET/$SERVICE_NAME/$DUMP_FILE', \
+    @s3_arn_to_backup_to='arn:aws:s3:::$BACKUP_BUCKET/$SERVICE_NAME-$DEPLOY_ENV/$DUMP_FILE', \
```

Comment:
I like the idea of having the env prefix as the first key in the namespace - `$BACKUP_BUCKET/$DEPLOY_ENV/$SERVICE_NAME/$DUMP_FILE` instead of `$BACKUP_BUCKET/$SERVICE_NAME-$DEPLOY_ENV/$DUMP_FILE` - might make things easier to find/filter on later on when looking for the backups - what do you think?